### DISCUSSION TITLE
Cloudflare error

### DISCUSSION THREAD

**Discussion by ririko5834** [NginxProxyManager/nginx-proxy-manager#1928] [Q&A]
When I have added proxy/redirection host without SSL, and proxied it with cloudflare I am getting error:
![image](https://user-images.githubusercontent.com/64729293/158020796-36a558c0-4c2d-4cef-8764-e03713d77c5b.png)

What can be the reason? I don't need let encrypt cert, when Cloudflare proxy auto creates cert.

When disabling cloudflare proxied state and visiting the domain with http://, it worked.

What can cause the error?

My cf setting:
![image](https://user-images.githubusercontent.com/64729293/158020827-968ca303-50d5-420a-9563-b9aed52bbca4.png)

**Comment by clemans** [comment]
+1 this issue. I'm having to revert to non-proxy DNS records for Cloudflare in order to get my services to work. Would love to get some feedback on how to fix this issue.

**Comment by ririko5834** [comment]
There should be an option to use Nginx proxy manager with Cloudflare proxy (proxied state on Cloudflare and without auto-generated lets encrypt SSL from npm)

**Comment by bernardatgh** [comment]
Are there any solutions to this issue, please? I tried using let's ssl certification but either Chrome or Edge showed the site reverse proxied by NPM in let'sencrypt SSL is unsecured. The site was still unsecured even by login NPM control panel to make use of the site with CF PEM and Key in custom mode. thanks,

**Reply by ririko5834** [reply]
Not sure
